lms/templates/courseware/course_about.html

```diff
@@ -228,7 +228,7 @@ <h1>
             <li class="important-dates-item">
               <span class="icon fa fa-calendar" aria-hidden="true"></span>
               <p class="important-dates-item-title">${_("Classes Start")}</p>
-              % if isinstance(course_start_date, str):
+              % if isinstance(course_start_date, (str, unicode)):
```

Comment:
a more correct check would be:
```
% if isinstance(course_start_date, basestring):
```
(from my PR: https://github.com/edx/edx-platform/pull/17263)

Reply:
https://docs.python.org/2/library/functions.html#basestring